django/contrib/gis/db/backends/postgis/pgraster.py

```diff
@@ -0,0 +1,161 @@
+import binascii
+import struct
+
+from django.forms import ValidationError
+
+from .const import (
+    GDAL_TO_POSTGIS, GDAL_TO_STRUCT, POSTGIS_HEADER_STRUCTURE, POSTGIS_TO_GDAL,
+    STRUCT_SIZE,
+)
+
+
+def pack(structure, data):
+    """
+    Packs data into hex string with little endian format.
```

Comment:
Lastest decision is to use pep8 style docstrings for new code. e.g. "Packs -> Pack"